exercises/practice/roman-numerals/cases_test.go

```diff
@@ -30,4 +30,5 @@ var romanNumeralTests = []romanNumeralTest{
 	{911, "CMXI", false},
 	{1024, "MXXIV", false},
 	{3000, "MMM", false},
+	{3999, "MMMCMXCIX", false},
```

Comment:
The `cases_test.go` files are automatically generated with our test generator, which in turn takes as reference the test cases specified in problem specs. For that reason, like in the previous change, tests to this file must also be discussed and added first in the problem spec repo.

Personally, I'm against adding this test right now. Since the instructions mention to not go higher than `3000`, most people I've seen solving this exercise logically do check if the value is less than `3000`. Adding this test will necessarily break all those solutions and will force people to revisit the exercise if they want to keep it up-to-date, which I think it's unnecessary for such a small change. Our exemplar solution also does this check, that's why CI is failing right now.

I would consider adding this test if the description of the problem would change too, but like previously mentioned, both the change to the description and the new test must be discussed in the problem spec repo.